internal/session/repository_session_test.go

```
@@ -1563,7 +1563,7 @@ func TestRepository_deleteTargetFKey(t *testing.T) {
 	iamRepo := iam.TestRepo(t, conn, wrapper)
 	kms := kms.TestKms(t, conn, wrapper)
 	ctx := context.Background()
-	repo, err := NewRepository(ctx, rw, rw, kms)
```

Same here, add a `require.NoError(err)` instead of ignoring this error.